src_python/habitat_sim/utils/datasets_download.py

```diff
@@ -191,6 +191,29 @@ def initialize_test_data_sources(data_path):
         }
     )
 
+    data_sources.update(
+        {
+            f"hm3d_{split}_semantics_v0.1": {
```

Comment:
If we update the version number below, the name does not have to be versioned.
```suggestion
            f"hm3d_{split}_semantics": {
```

Reply:
> I think we need to make a choice about versioning here.
> 
> If we want to host multiple versions simultaneously then we'll need separate entries with version in the name. However, if we intend to update the versions sequentially without maintaining download references to previous versions, then we should remove the versioning from the names and let the `"version":` tag and links be the only dynamic property here.


It might be worth maintaining multiple versions simultaneously for the sake of experimental reproducibility.